staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/errors/statuserror.go

```diff
@@ -32,6 +33,15 @@ func ToStatusErr(webhookName string, result *metav1.Status) *apierrors.StatusErr
 		result = &metav1.Status{Status: metav1.StatusFailure}
 	}
 
+	// Make sure we don't return < 400 status codes along with a rejection
+	if result.Code < http.StatusBadRequest {
+		result.Code = http.StatusBadRequest
+	}
+	// Make sure we don't return "" or "Success" status along with a rejection
+	if result.Status == "" || result.Status == metav1.StatusSuccess {
```

Comment:
might be a stupid question: can the webhook return a status.Status w/ other random string "FooBarABC"?

Reply:
> if it's set to something other than "Success" or "Failure", I'm really hesitant to mess with it

oh i missed it